handout/components/structural_directives.md

```diff
@@ -1,16 +0,0 @@
-# Structural Directives
-
-Angular's structural directives change the DOM layout by adding and removing DOM elements. For example:
-
-```html
-<div *ngIf="hero">{{ hero }}</div>
-
-<div *ngFor="let hero of heroes">{{ hero }}</div>
-
-<div [ngSwitch]="status">
-  <template *ngSwitchCase="'in-mission'">In Mission</template>
-  <template *ngSwitchCase="'ready'">Ready</template>
-  <template *ngSwitchDefault>Unknown</template>
-</div>
-```
-
```

Comment:
The file used in the book, is located under the `directives` folder:
`handout/directives/structural_directives`
This one under `components` folder, had no reference in the main markdown file: `SUMMARY.md`